src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -109,9 +109,12 @@ class ClaimSimulation extends Simulation {
       .formParam("_csrf", "${csrf_token}")
     )
 
-      .exec(http("submit")
```

Comment:
We no longer post to the check page, the test's will get it as they will follow redirects.